src/js/steps/letters/init.js

```diff
@@ -41,7 +41,7 @@ module.exports = function ($events, options) {
       }
     });
 
-    $letters.on('click', '.letter', function () {
+    $letters.on('click', '.letter:not(.nonclickable)', function () {
```

Comment:
Why have two classes to do the same thing? Why not just not `:not(.special-char)`?